dist/jquery.panzoom.js

```diff
@@ -1160,9 +1169,6 @@
 			// Remove any transitions happening
 			this.transition(true);
 
-			// Indicate that we are currently panning
-			this.panning = true;
-
```

Comment:
This seems to be because of the new Pan sensitivity functionality.. Move to a different PR, please.